recipe-server/client/selfrepair/tests/test_normandy_driver.js

```diff
@@ -103,6 +104,35 @@ describe('Normandy Driver', () => {
       expect(client.isDefaultBrowser).toEqual(true);
       expect(client.searchEngine).toEqual('Yahoo');
     });
+
+    it('should pull plugin data from navigator.plugins', async () => {
+      const flashPlugin = {
+        name: 'Shockwave Flash',
+        description: 'Literally the worst',
```

Comment:
Much as I sympathize and hate to be the killjoy person in the room, we should probably use a less... opinionated description. :-)